src/components/MenuPanel/OrganizationSwitcher/OrganizationItem.js

```diff
@@ -22,22 +22,7 @@ class OrganizationItem extends React.Component {
         `}
         {...props}
       >
-        <OrgIcon rounded={!knownOrg}>
-          {knownOrg ? (
-            <img
-              src={knownOrg.image}
-              width={6 * GU}
```

Comment:
These items used to "say" they were 6 GU, but were in reality only 3 GU, and the designs are also 3.